project/api/views.py

```diff
@@ -112,6 +119,30 @@ class SpacetimeVolumeViewSet(viewsets.ModelViewSet):
     serializer_class = SpacetimeVolumeSerializer
 
 
+class NarrativeVoteViewSet(viewsets.ModelViewSet):
+    """
+    Viewset for NarrativeVote model
+    """
+
+    queryset = NarrativeVote.objects.all()
+    serializer_class = NarrativeVoteSerializer
+    permission_classes = (IsAuthenticatedOrReadOnly, IsUserOrReadOnly)
+
+    def create(self, request, *args, **kwargs):
+        """
+        Deletes instance if vote is null
+        """
+
+        request.data["user"] = request.user.id
```

Comment:
Probably we could update the NarrativeVotes model and manage it the same way as Profile
https://github.com/chronhq/backend/blob/bf063e1b37d7dac37e9e2e2531ac5c7913538a67/project/api/models.py#L64-L65